port/zos390/omrgetthent.h

```diff
@@ -28,7 +28,7 @@
 #define PGTHA_ACCESS_CURRENT 1
 #define PGTHA_FLAG_PROCESS_DATA 0x80
 
-#pragma pack(packed)
+#pragma pack(push, 1)
```

Comment:
Does this mean the same for both xlc and OpenXL?

Reply:
yes they should be functionally the same.
https://www.ibm.com/docs/en/zos/2.4.0?topic=descriptions-pragma-pack
also have confirmed internally with Sean from the open-xl/cpp team

Reply:
Have changed this to `#pragma pack(1)` after having confirmed the packed struct sizes between XLC and Open XL and confirming the functionality is identical with Sean. He suggested to use this notation instead.